spec-main/fixtures/crash-cases/early-in-memory-session-create/index.js

```diff
@@ -2,5 +2,7 @@ const { app, session } = require('electron');
 
 app.on('ready', () => {
   session.fromPartition('in-memory');
-  process.exit(0);
+  setImmediate(() => {
+    process.exit(0);
+  });
```

Comment:
it's weird that this crashes on woa but not elsewhere 🤔

Reply:
indeed